Dockerfile.dev

```diff
@@ -0,0 +1,23 @@
+FROM ubuntu:bionic
+
+RUN useradd -m retdec
+WORKDIR /home/retdec
+ENV HOME /home/retdec
+
+RUN apt-get -y update && \
+    DEBIAN_FRONTEND=noninteractive apt-get install -y build-essential git bc graphviz upx cmake python zlib1g-dev flex bison libtinfo-dev autoconf pkg-config m4 libtool wget
+
+COPY . retdec
+RUN chown -R retdec:retdec retdec
```

Comment:
I was going to use the `--chown` flag, but my docker version didn't support it. I wasn't able to find out what docker version comes with Ubuntu. If you're comfortable with the docker version dependency I'll change this to the `--chown` flag.

Reply:
The 17.x Docker release is pretty new, and ships only with the upcoming Ubuntu 18.04 (available in April next year). I guess that if we used the `--chown` flag, a lot of users would complain that they are unable to build the image. Therefore, at the end, I think we should keep the current version that uses a standalone `chown`. In this way, the Dockerfile can be used by a wider range of users.

Nevertheless, maybe we should add the `COPY --chown` version at least in a comment with a note? Or is there a way of writing a conditional based on the Docker version?

Reply:
As far as I understand, you can't conditionally run different docker commands (RUN, COPY etc). 
I'll add the --chown command as a comment, perhaps in the future it will be more reasonable to use that version.